src/app/project/template.html

```diff
@@ -209,7 +209,10 @@
            class="km-health-state km-project-cards-status"
            fxFlexAlign=" center"></i>
         <mat-card-title [matTooltip]="getProjectTooltip(project.name)"
-                        [attr.id]="'km-project-name-' + project.name">{{getName(project.name)}}</mat-card-title>
+                        [attr.id]="'km-project-name-' + project.name"
+                        class="card-header">
```

Comment:
I do not see this `card-header` class definition. Where is it?

Reply:
It's a leftover. PTAL